### ISSUE TITLE
When manual is set to false, script shouldn't fail on missing manual_file

### ISSUE THREAD

**Issue by gwarf** [EGI-Foundation/glite-info-update-endpoints#17]
When `manual` is set to `False` in `etc/glite-info-update-endpoints.conf` and when `manual_file` is not set (ie. both paramater are using default values) the script fails:

```
2018-11-26 16:40:14,165 ERROR    Missing parameter 'manual_file' in the configuration file etc/glite-info-update-endpoints.conf.
```

**Comment by brucellino** [comment]
This sounds like a case of bad defaults. 

It sounds like it should also be done in the python script, no?

**Comment by gwarf** [comment]
OK, so my bad, I broke it here: https://github.com/EGI-Foundation/glite-info-update-endpoints/commit/4dd8ae952a68118b8e9918ff5c1e6cbc19e9663b?diff=unified#diff-4cacda860b02b2a0bb9d5be62b7533eeR5
...
Do you want to make a PR for it? Or should I?

**Comment by gwarf** [comment]
Fixed by #19
